doc/specs/portable-mode-spec.md

```diff
@@ -0,0 +1,90 @@
+---
+author: Dustin L. Howett @DHowett
+created on: 2023-03-22
+last updated: 2023-03-22
+issue id: none
+---
+
+# Windows Terminal "Portable" Mode
+
+## Abstract
+
+Since we are planning on officially supporting unpackaged execution, I propose a special mode where Terminal stores its
+settings in a `settings` folder next to `WindowsTerminal.exe`.
+
+## Inspiration
+
+- [PortableApps](https://portableapps.com)
+- "Embeddable" Python, which relies on the deployment of a specific file to the Python root
+
+## Solution Design
+
+- _If running without package identity,_ `CascadiaSettings` will look for the presence of a file called `.portable` next
+  to `Microsoft.Terminal.Settings.Model.dll`.

```

Comment:
Just for clarity - is there anything in this file (aside from the future consideration of having a directory path to the settings)? Or is its presence just acting as a flag

Reply:
Presence alone!